ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,388 @@
+# ERC-X: Consolidated Dynamic Storage in Solidity  
+
+## Title: Consolidated Dynamic Storage
```

Comment:
Please fix the formatting for the header.
title:  
description:  
author: 
discussions-to: 
status: Draft
type: Standards Track
category: ERC
created: 
requires: